trivia.js

```diff
@@ -0,0 +1,203 @@
+var correctAnsId
+
+// Start game function
+$(document).ready(function () {
+  $('.container').hide()
+  $('.answerTitle').hide()
+  $('.nextQuestion').hide()
+  startGame()
+})
+
+function startGame () {
+  $('#startGame').on('click', function () {
+    $('#startGame').hide()
+    $('.container').show()
+    $('.timer').html(`00:${time}`)
+    runTimer()
+    display()
+  })
+}
+
+//Display question & answers
+let idArr = ['#A', '#B', '#C', '#D']
+function display() {
+  $('.questionTitle').append(trivia[questionCounter - 1].question)
+  for (let i = 0; i < trivia[questionCounter - 1].answers.length; i++) {
+    $(idArr[i]).append(trivia[questionCounter - 1].answers[i])
+    if (trivia[questionCounter - 1].answers[i] === trivia[questionCounter - 1].correctAns) {
+      correctAnsId = idArr[i]
+    }
+  }
+}
+
+function empty () {
+  $('.questionTitle').empty()
+  $('.message').empty()
+  $('.choices').prop('checked', false)
+  $('#A').empty()
+  $('#B').empty()
+  $('#C').empty()
+  $('#D').empty()
+}
+
+function checkAns () {
+  var checkAns = $(this).siblings('span').text()
+  checkAns = checkAns.split(': ')
+  // this is supposed to give the answers an id
+  if (checkAns[1] === trivia[questionCounter - 1].correctAns) {
+    questionCounter++
+    correctAns++
+  } else {
+    questionCounter++
+    incorrectAns++
+  }
+  result()
+  $('.nextQuestion').show()
+}
+
+function result () {
+  $('.questionTitle').parent().hide()
+  $('.answerTitle').show()
+  for (let i = 0; i < trivia[questionCounter - 1].answers.length; i++) {
+    if (idArr[i] !== correctAnsId) {
+      $(idArr[i]).parent().hide()
+    }
+  }
+}
+
+function nextQuestion () {
+  empty()
+  display()
+  showNew()
+  // this is supposed to reset the timer but doesnt work as well
+  let time = 30
+  runTimer()
+}
+
+function endGame () {
+  let 
+  if (questionCounter = 10 ) {
+    //i couldnt get checkAns() to give the answer array an id so this is supposed to display correctAns, incorrectAns, and unanswered and display them
+  }
+}
+
+// Shows next question and answers
+function showNew () {
+  $('.questionTitle').parent().show()
+  for (let i = 0; i < trivia[questionCounter - 1].answers.length; i++) {
+    $(idArr[i]).parent().show()
+  }
+  $('.answerTitle').hide()
+  $('.nextQuestion').hide()
+}
+
+//Event Listener
+$('.choices').on('click', function () {
+  stop()
+  checkAns()
+})
+$('.nextQuestion').on('click', function () {
+  $('.choices').removeAttr('checked')
+  nextQuestion()
+})
+
+function runTimer () {
+  clearInterval(intervalId);
+  intervalId = setInterval(decrement, 1000)
+}
+
+function decrement() {
+  time--;
+  $('.timer').text("00:" + converter(time));
+  if (time === 0) {
+    stop();
+    $('.timer').parent().text("Time's Up")
+    unanswered++
+    result()
+  }
+  nextQuestion()
+}
+
+function converter(timer) {
+  if (timer < 10) {
+    timer = '0' + timer
+  }
+  return timer
+}
+
+function stop() {
+  clearInterval(intervalId);
+}
```

Comment:
good job overall! this is definitely an app that you can refactor later when you get more comfortable with functions and ES6.